src/src/rx_main.cpp

```diff
@@ -457,96 +525,119 @@ void ICACHE_RAM_ATTR sampleButton()
 
     if ((millis() > buttonLastPressed + BUTTON_RESET_INTERVAL) && buttonDown)
     {
-#ifdef PLATFORM_ESP8266
+        #ifdef PLATFORM_ESP8266
         ESP.restart();
-#endif
+        #endif
     }
 
     buttonPrevValue = buttonValue;
 }
 
+void ICACHE_RAM_ATTR RXdoneISR()
+{
+    ProcessRFPacket();
+    //Serial.println("r");
+}
+
+void ICACHE_RAM_ATTR TXdoneISR()
+{
+    alreadyTLMresp = false;
+    addPacketToLQ(); // Adds packet to LQ otherwise an artificial drop in LQ is seen due to sending TLM.
+    Radio.RXnb();
+}
+
 void setup()
 {
+    delay(100);
+    Serial.println("ExpressLRS Module Booting...");
+
 #ifdef PLATFORM_STM32
     Serial.setTx(GPIO_PIN_RCSIGNAL_TX);
     Serial.setRx(GPIO_PIN_RCSIGNAL_RX);
-    Serial.begin(420000);
```

Comment:
Ummmm, we need this for MSP...

Reply:
I just moved it lower, there was some duplication going on. I don't think it broke MSP?